Keas.Mvc/Controllers/Api/SpacesController.cs

```diff
@@ -24,44 +23,44 @@ public async Task<IActionResult> SearchSpaces(string q)
         {
             var queryWords = q.ToLower().Split(" ").ToList();
             var space = await _context.Spaces
-                .Where(x => (!String.IsNullOrWhiteSpace(x.BldgName) && queryWords.Any(s => x.BldgName.ToLower().Contains(s)))
-                || (!String.IsNullOrWhiteSpace(x.RoomName) && queryWords.Any(s => x.RoomName.ToLower().Contains(s)))
-                || (!String.IsNullOrWhiteSpace(x.RoomNumber) && queryWords.Any(s => x.RoomNumber.ToLower().Contains(s))))
+                .Where(x => (!string.IsNullOrWhiteSpace(x.BldgName) && queryWords.Any(s => x.BldgName.ToLower().Contains(s)))
+                || (!string.IsNullOrWhiteSpace(x.RoomName) && queryWords.Any(s => x.RoomName.ToLower().Contains(s)))
+                || (!string.IsNullOrWhiteSpace(x.RoomNumber) && queryWords.Any(s => x.RoomNumber.ToLower().Contains(s))))
                 .AsNoTracking().ToListAsync();
             return Json(space);
         }
 
-        public async Task<IActionResult> List(string orgId)
+        public async Task<IActionResult> List(string teamSlug)
```

Comment:
@jcubbage if you just take team name as a param here it should map properly, and then you can get rid of that teamSlug query string param you added